cmake/mcuboot.cmake

```diff
@@ -16,6 +16,19 @@ function(zephyr_runner_file type path)
   set_target_properties(runners_yaml_props_target PROPERTIES "${type}_file" "${path}")
 endfunction()
 
+function(zephyr_mcuboot_hex_bin_convert hex_file bin_file)
```

Comment:
This is not a transferable way of doing this and requires use of this specific file to achieve this (and this file can now be completely replaced out of tree), would rather something generic is added to the cmake code that can convert any hex to bin (or just use objcopy) and can be called as is needed